detections/endpoint/executables_or_script_creation_in_suspicious_path.yml

```
@@ -1,7 +1,7 @@
 name: Executables Or Script Creation In Suspicious Path
 id: a7e3f0f0-ae42-11eb-b245-acde48001122
-version: 16
-date: '2025-05-06'
+version: '17'
```

@tccontre This is a comment for all the rules you changed. Please do not use the single quote in the version field
```suggestion
version: 17
```